src/components/account/fission.ts

```diff
@@ -306,9 +307,11 @@ export async function did(
 ////////
 
 export function implementation(
-  endpoints: Fission.Endpoints,
-  dependencies: Dependencies
+  dependencies: Dependencies,
+  optionalEndpoints?: Fission.Endpoints
 ): Implementation<Annex> {
+  const endpoints = optionalEndpoints || Fission.PRODUCTION
```

Comment:
All Fission components now default to production env, instead of having multiple files that you could import.